Shapes.java

```diff
@@ -1,25 +1,77 @@
-import java.util.*;
-
-public class Shapes{
+import java.util.Scanner;
+public class Shapes {
   public static void main(String[] args) {
+    areaRectangle();
+    areaTriangle();
+    areaCircle();
+    areaTrapezoid();
+
+  }//end main method
+
+  public static void areaRectangle(){
+    Scanner input = new Scanner(System.in);
+    int length = 0;
```

Comment:
nice variable names! They clearly tell whoever's reading your code what they represent and their purpose. Also great job making the all `camelCase`.